benchto-driver/src/main/java/com/teradata/benchto/driver/listeners/GraphiteEventExecutionListener.java

```diff
@@ -37,9 +38,18 @@
     private TaskExecutor taskExecutor;
 
     @Autowired
+    private ExecutionSynchronizer executionSynchronizer;
+
+    @Autowired
     private GraphiteClient graphiteClient;
 
     @Override
+    public int getOrder()
```

Comment:
why order is required now since listeners are not async and https://github.com/prestodb/benchto/pull/25/commits/287c36cc29f1d994db187a218836950ec7d0b0e2#diff-5dce60daacbd65fc59e4ebfc7394dc57R41 is still valid?